src/app/components/AccountMenu/index.tsx

```diff
@@ -130,7 +130,8 @@ function AccountMenu({ showOptions = true }: Props) {
             <WalletIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
             {t("options.account.account_settings")}
           </Menu.ItemButton>
-          {isAlbyAccount(authAccount?.alias) && (
+          {(isAlbyLNDHubAccount(authAccount?.alias) ||
```

Comment:
Draft PR for taking avatar image from Alby account: https://github.com/pavanjoshi914/lightning-browser-extension/pull/4